bin/stripdata.js

```diff
@@ -121,6 +127,31 @@ function splitDeviceId(deviceId) {
 	return retlist;
 }
 
+function removeIDsAndPayload(data) {
+	if (program.stripAll) {
+		if (data.payload)
+			delete data.payload;
+		if (data.id)
+			delete data.id;
+		if (data.guid)
+			delete data.guid;
+		if (data.bolus)
```

Comment:
Do we always want to remove `data.bolus`, or only under certain circumstances (for example, `type` is `wizard`)? Even if `data.bolus` appears only in `wizard` types currently, I think it would be good to qualify in case it causes unexpected behaviour in future.